src/apps/reservation-list/modal/pause-reservation/pause-reservation.tsx

```diff
@@ -32,25 +32,31 @@ const PauseReservation: FC<PauseReservationProps> = ({ id, user }) => {
   const [endDate, setEndDate] = useState<string>("");
 
   const save = useCallback(() => {
-    if (user && startDate && endDate) {
+    if (user) {
+      const saveData = {
+        preferredPickupBranch: user.preferredPickupBranch,
+        receiveEmail: user.receiveEmail,
+        receivePostalMail: user.receivePostalMail,
+        receiveSms: user.receiveSms
+      } as Patron;
+
+      if (startDate || endDate) {
+        saveData.onHold = {
+          from: startDate === "" ? undefined : startDate,
+          to: endDate === "" ? undefined : endDate
+        };
+      }
```

Comment:
I have mentioned it before but I am not going to make a fuzz and block it. But we have I think 3 places (maybe more) where we save patron data. It should be possible to have a common function for it. Displeased old man shuts up now.

Reply:
Yeah, we've talked about this, they are similar but a little different in the default settings, e.g. whether to save recieveSMS default as true/false. I'll add a todo, because they could be merged, but I do not have the overview of the functionality in the different places.